terraform/modules/services/api/eb.tf

```diff
@@ -60,6 +60,12 @@ resource "aws_elastic_beanstalk_environment" "eb-api-env" {
   }
 
   setting {
+    namespace = "aws:elasticbeanstalk:application:environment"
+    name      = "AUTH0_RULE_NAMESPACE"
+    value     = "https://openclimatefix.org"
+  }
+
+    setting {
```

Comment:
pre-commit missed the misaligned indentation here potentially?

Reply:
im not sure there is a pre-commit here. We should think about putting one in, I think theres a `terraform lint` command, or something like that